index.js

```diff
@@ -10,30 +11,36 @@ const getHtmlFiles = async (directory) => {
   return files.map((file) => file.fullPath)
 }
 
+const CRITICAL_CONCURRENCY_LIMIT = 4
```

Comment:
I’m not sure what a good number is 😅

Reply:
Some libraries use `os.cpus().length` for default concurrency limit. Others (like Ava) add a check that decreases this to `2` when CI is detected.

Reply:
`async-pool` doesn’t handle forking to a new core, and running multiple puppeteer instances in the same process is not a problem, so I’d need to rework this, but I think I’ll go with an hardcoded 4 for now, I’ll decrease to 2 if the error still happens.